src/lib.rs

```diff
@@ -17,3 +25,30 @@ pub mod primes;
 #[allow(deref_nullptr)]
 #[cfg(all(not(target_arch = "wasm32"), not(windows), feature = "libsodium"))]
 pub mod sodium_bindings;
+
+#[derive(Debug, Error)]
+pub enum Error {
```

Comment:
let's create an `error.rs` file for this

Reply:
ok